scripts/phrase_context_classifier.py

```diff
@@ -29,12 +29,12 @@ def __init__(self, embedding_dim, hidden_size, num_layers, forward_hidden_dim, l
         self._hidden_layer = nn.Linear(2 * embedding_dim + hidden_size * 2, forward_hidden_dim)
         self._output_layer = nn.Linear(forward_hidden_dim, label_nr)
 
-    def forward(self, word1, word2, context, context_lengths, training, device):
+    def forward(self, batch, training, device):
         # context size = batchsize x max len x embedding dim
         # convert the padded context into a packed sequence such that the padded vectors are not shown to the LSTM
         # context_packed = sum of all seq lenghts, embedding_dim
         # batch_sizes = column-wise (how many real elements do I have?)
-        context_packed = nn.utils.rnn.pack_padded_sequence(context, context_lengths, batch_first=True,
```

Comment:
maybe we can add an assert statement here to ensure that "seq" and "seq_lengths" are in the dictionary ..
assert "seq" in batch.keys() and "seq_lengths" in batch.keys(), "no sequence and sequence length in bacht" o.ä dann weiß man gleich dass man das falsche Datenset geladen hat